lib/plugins/aws/deploy/compile/events/apiGateway/tests/methods.js

```diff
@@ -162,21 +162,25 @@ describe('#compileMethods()', () => {
   });
 
   it('should set the correct lambdaUri', () => {
-    const lambdaUri = `arn:aws:apigateway:${
-      awsCompileApigEvents.options.region}:lambda:path/2015-03-31/functions/arn:aws:lambda:${
-      awsCompileApigEvents.options.region}:12345678:function:${
-      awsCompileApigEvents.serverless.service.service}-${
-      awsCompileApigEvents.options.stage}-first/invocations`;
+    const lambdaUriObject = {
+      'Fn::Join': ['', [
+        'arn:aws:apigateway:',
+        { Ref: 'AWS::Region' },
+        ':lambda:path/2015-03-31/functions/',
+        { 'Fn::GetAtt': ['first', 'Arn'] },
+        '/invocations'  // eslint-disable-line comma-dangle
+      ]]                // eslint-disable-line comma-dangle
```

Comment:
@pmuens I couldn't get eslint to parse this correctly, I'm not sure how to fix this correctly without disabling the eslint parser for these lines. Eslint wants me to add comma's here, but when I add them (which don't make sense), it wants me to delete them ;) Perhaps the notation is wrong?